### ISSUE TITLE
kodo doesn't see notifs in the notif tab since aug 28

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#2679]

**Comment by ara4n** [comment]
i assume this is because he's configures some weird serverside notif settings

**Comment by ara4n** [comment]
@tjgillies are you still seeing this?

**Comment by tjgillies** [comment]
@ara4n yeah still haven't seen anything new. I've tried toggling all my settings.

> i assume this is because he's configures some weird serverside notif settings  

I haven't done anything like that, that I can recall

**Comment by tjgillies** [comment]
This is happening on another identity now. kodo@tyler.cat

**Comment by SimonBrandner** [comment]
@hackervera, are you still seeing this?
